awx/ui_next/src/RootProvider.jsx

```diff
@@ -15,10 +15,10 @@ class RootProvider extends Component {
     const language = getLanguageWithoutRegionCode(navigator);
 
     return (
-      <HashRouter>
+      <HashRouter basename="/next">
```

Comment:
I noticed the URL ends up looking like `next/#next/users` with `next` both before and after the hash. Do we need it after the hash also?

Reply:
That is how the front end routing system and back end got connected. Django is routing any URL that has `next` to the template `index.html`. The `index.html` has the link for the static files from the `npm run build`. Then frontend routing system now has this keyword `next` to be connected to an unique URL in backend always serving the same file. I think if we use `BrowserRouter` instead of the `HashRouter` we will not have this issue, probably.  But this was a temporary solution.

Reply:
@nixocio @keithjgrant I think we just need to change `basename` --> `baseName` and this should work with just the one leading `/next`:

![ui-next-integration-01](https://user-images.githubusercontent.com/9753817/82675102-4fc6e280-9c12-11ea-9577-64e65d2db8d4.gif)

Reply:
On second look, basename is the correct prop name so perhaps we don't need it? man_shrugging works for me if that is the case.

Reply:
You are right @jakemcdermott . I removed the `basename` and just worked fined. Thanks for verifying this.

Reply:
Added `basenname` back in after switching to `BrowserRouter`